packages/phoenix-evals/src/phoenix/evals/models/mistralai.py

```diff
@@ -161,8 +151,8 @@ async def _async_rate_limited_completion(self, **kwargs: Any) -> Any:
         @self._rate_limiter.alimit
         async def _async_completion(**kwargs: Any) -> Any:
```

Comment:
This function returns `response.choices[0].message.content`

This is no longer typed correctly it looks like

first, check if response exists before accesing choices, then check that choices is a list etc

```
if response and choices := response.choices:
    if main_response := choices[0]:
        return main_response.message.content
```

Reply:
Updated!